packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -283,17 +284,51 @@ gmaps.Icon? _gmIconFromBitmapDescriptor(BitmapDescriptor bitmapDescriptor) {
         ..size = size
         ..scaledSize = size;
     }
+  } else if (iconConfig[0] == 'asset') {
```

Comment:
The implementations of the new `asset` and `bytes` are very similar to the existing ones. Any possibility to unify them?